linter/whitelists/fr.txt

```diff
@@ -0,0 +1,38 @@
+Avec
+Avis
+Cela
+afficher
+ajouter
+avis
+ce
+copyright
+d'archives
+dans
+de
+des
+devez
+différents
+est
+et
+facilement
+fournisseurs
+images
+les
+licence
+module
+notice
+pack
+particulièrement
+photos
+pour
+pouvez
+que
+si
+stock
+toutes
+un
+une
+utile
+utilisez
+vous
+à
```

Comment:
A lot of these words shouldn't really needed to be whitelisted, should they? 😁

Reply:
I dont know, i put all words i used in my .yml in this file.

Reply:
Well, the point of the whitelist is not to put _all_ words in there. The spell checker is there to help spelling errors. If you add all words (including words with spelling errors for example) then it is of not much help ;). Why did you add all words?

Reply:
Because that is my best french and i dont see an error. i hate this linter shit and have no just to resolve one error, commit,  and another, commit, and another, commit. i have things to do. 

Why is that so important that we spellcheck the descriptions? nobody cares, everyone is self responsible for this. thats my opinon.

Reply:
One of the first error where "there is noch fr.txt". And then, i added one. done.

Reply:
when i use in future the translator for nl, ru, pl, how the hack can i say that is correct typo? its good enough for the user and that is wat matters.